modules/social_features/social_branding/modules/social_branding_required_test/social_branding_required_test.info.yml

```
@@ -0,0 +1,8 @@
+name: 'Social Branding required test'
+description: 'Support module for module Social Branding testing.'
+type: module
+core: 8.x
+core_version_requirement: ^8 || ^9
+dependencies:
+  - social:social_branding
+package: Social (Experimental)
```

Let's make this

```suggestion
package: Testing
hidden: true
```

Just to be clear this is for testing only and ensure it doesn't accidentally show up anywhere anyway :)